packages/block-editor/src/components/block-navigation/style.scss

```diff
@@ -274,7 +274,34 @@
 	}
 }
 
-.block-editor-block-navigator-indentation {
-	flex-shrink: 0;
-	width: $grid-unit-30 + $grid-unit-05; // Indent a full icon size, plus 4px which optically aligns child icons to the text label above.
+// First level of indentation is aria-level 2, max indent is 8.
```

Comment:
Should we use a SCSS for loop for this one?

Reply:
Good call I added this in https://github.com/WordPress/gutenberg/pull/32063/commits/e5c2be34da74e3c029d886308892f97fde3efc1a

Generated final CSS looks like:
<img width="512" alt="Screen Shot 2021-05-21 at 8 57 24 AM" src="https://user-images.githubusercontent.com/1270189/119166818-ef7ca380-ba13-11eb-86f9-678368f86a0f.png">